File: GeneratedEntityReaderClasses/Classes/PathFindingJumpParams.cs
namespace GeneratedNoitaClasses;
public class PathFindingJumpParams :  iNoitaType<PathFindingJumpParams>{
	public NoitaFloat x { get; set; } = new();
	public NoitaFloat y { get; set; } = new();
	public NoitaFloat lob { get; set; } = new();

    public void Read(NoitaStream s){
		x.Read(s);
		y.Read(s);
		lob.Read(s);

    }
    public void Write(NoitaStream s){
		x.Write(s);
		y.Write(s);
		lob.Write(s);

    }
}

File: GeneratedEntityReaderClassesPython/GeneratedClasses/PathFindingJumpParams.py
from dataclasses import dataclass, field
from BasicNoitaTypes import NoitaFloat

from NoitaStream import NoitaStream
@dataclass
class PathFindingJumpParams:
    x:NoitaFloat = field(default_factory=NoitaFloat)
    y:NoitaFloat = field(default_factory=NoitaFloat)
    lob:NoitaFloat = field(default_factory=NoitaFloat)
    
    def Read(self, s:NoitaStream):
        self.x.Read(s)
        self.y.Read(s)
        self.lob.Read(s)
    
    def Write(self, s:NoitaStream):
        self.x.Write(s)
        self.y.Write(s)
        self.lob.Write(s)

